.yamato/_triggers.yml

```diff
@@ -18,6 +18,14 @@ pull_request_trigger_legacy:
   name: Pull Request Trigger on 2021.1 (master, develop, & release branches)
   dependencies:
     - .yamato/_triggers.yml#pull_request_trigger
+  triggers:
+    cancel_old_ci: true
+    pull_requests:
+    - targets:
+        only:
+          - "master"
+          - "develop"
+          - "/release\/.*/"
```

Comment:
I'm assuming these triggers are "or'ed", so that it's either pull requests or specific branches? Or is it pull requests to these specific branches?

Reply:
It's only PRs in these specific branches. This is actually copied from the trigger below so that it's the exact same trigger conditions